### PR TITLE
Improves style and titles of toolbar items in notebooks.

### PR DESCRIPTION
**Pull request by jbicker**
#### What it does

As adressed in https://github.com/eclipse-theia/theia/issues/12863 it improves the style and tooltips of the toolbar items in notebooks.

#### How to test

Follow the "How to test" instructions [here](https://github.com/eclipse-theia/theia/pull/12442) and hover toolbar items and experience the amazing look of rounded corners and see that the tooltips are same as in VSCode.   

#### Review checklist

- [x] As an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

File: packages/notebook/src/browser/contributions/notebook-actions-contribution.ts [hunk 1]
```diff
@@ -59,6 +59,12 @@ export namespace NotebookCommands {
     });
 }
 
+// for commands where the html title differs from button label
+export const NOTEBOOK_COMMAND_TITLES: {[key: string]: string} = {
+    'notebook.add-new-markdown-cell': nls.localizeByDefault('Add Markdown Cell'),
+    'notebook.add-new-code-cell': nls.localizeByDefault('Add Code Cell')
+};
```
Comment (msujew) [inline packages/notebook/src/browser/contributions/notebook-actions-contribution.ts:66]: Suggestion: This doesn't seem like an extensible way of setting tooltips for specific commands.

[vscode simply uses](https://github.com/microsoft/vscode/blob/8ba046adbeecbbabac874d1a9b6b1a41f8cedf19/src/vs/workbench/contrib/notebook/browser/controller/insertCellActions.ts#L320-L325) a `tooltip` property on their commands which is displayed in the toolbar tooltip. We should probably do something similar.

Comment (jbicker) [reply packages/notebook/src/browser/contributions/notebook-actions-contribution.ts:66]: I was actually thinking about enhancing the command interface but then I thought it is maybe too much just for one case. But ok. Will do so.

Comment (msujew) [reply packages/notebook/src/browser/contributions/notebook-actions-contribution.ts:66]: It doesn't necessarily need to adjust the command interface. We have added special interfaces for that [in the past](https://github.com/eclipse-theia/theia/blob/9a51317fe81a179e9e7c6c9071e9a00218bba299/packages/notebook/src/common/notebook-common.ts#L25).

File: packages/notebook/src/common/notebook-common.ts [hunk 2]
```diff
@@ -19,7 +19,7 @@ import { MarkdownString } from '@theia/core/lib/common/markdown-rendering/markdo
 import { BinaryBuffer } from '@theia/core/lib/common/buffer';
 import { UriComponents } from '@theia/core/lib/common/uri';
 
-export interface NotebookCommand {
+export interface NotebookCommand extends Command {
     id: string;
```
Comment (msujew) [inline packages/notebook/src/common/notebook-common.ts:23]: Minor: The `id` property is no longer necessary

**Review by msujew — APPROVED** [review]
Thanks, looks good to me 👍

**Pull request merged**